<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>719</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Product $ Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>75</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="search_query_textbox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>34</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Enter Product URL or Search Query</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>34</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Relevance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Price: High to Low</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Price: Low to High</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="search_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>781</width>
         <height>324</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="console_log">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>fdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadffdfadsfadfadfadfadf</string>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
      <property name="centerOnScroll">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionShow_Logs"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShow_Logs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Logs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
